src/index.js

```diff
@@ -50,20 +54,55 @@ const hunt = () => {
   return enemy.msgs.encounter;
 };
 
+/**
+ * @param {number} low The lower bound (inclusive)
+ * @param {number} high The upper bound (exclusive)
+ * @returns {number} A random number between low and high
+ */
+const randomRange = (low, high) => Math.random() * (high - low) + low;
+
+/**
+ * @param {number} low The lower bound (inclusive)
+ * @param {number} high The upper bound (inclusive)
+ * @returns {number} A random integer between low and high
+ */
+const irandomRange = (low, high) => Math.floor(randomRange(low, high + 1));
+
+/**
+ * @returns {number} a random number between 0.8 and 1.2
+ */
+const random20 = () => 1 + irandomRange(-20, 20) / 100;
+
+/**
+ * @param {number[]} list A list of numbers
+ * @returns {number} the sum of numbers in the list
+ */
+const sum = (list) => list.reduce((total, value) => total + value, 0);
```

Comment:
Reduce is handy, but a bit tricky at first. It is equivalent to writing this but on one line:
```javascript
let total = 0;
for (const value of list) {
  total = total + value;
}
return value;
```